taichi/gui/gui.h

```diff
@@ -506,6 +506,8 @@ class GUI : public GUIBase {
   std::vector<std::unique_ptr<float>> widget_values;
   bool show_gui;
   bool fullscreen;
+  bool fast_gui;
+  size_t fast_buf;
```

Comment:
nit: Could you use `char *` (or `intptr_t`?) I initially thought this was the size of the buffer.

Reply:
But we can't pass `char *` in pybind11, isn't it? Let's use `uintptr_t` then.

Reply:
```suggestion
  uintptr_t fast_buf;
```